crawlerapp/frontend/app/components/d3-visualization.js

```diff
@@ -26,25 +30,32 @@ export default Component.extend({
             .append('g')
             .attr('class', 'nodes')
 
-        //Label - DO WE WANT A LABEL WITH THE TOOLTIP TOO???
-        //Uncomment to add label to each node, does not always look the best
-        // node.append('text')
-        //   .attr('x', 25)
-        //   .attr('y', 3)
-        //   .text(d => d.id);
-
         //Draw Circle for each Node
         let circles = node.append('circle')
             .attr('r', 12)//Create circle with radius size
-            .attr('fill', 'blue')
+            .attr('fill', function(d){
+
+                if(data.keywordFound && (data.keywordNode === d.id)){
+                    console.log('Keyword Node: ', d.id, ' - Keyword: ', data.keyword)
+                    return '#F1FA72' //Yellow
+                }
+                else if(d.depth === 0){
+                    console.log('Search Node: ', d.id)
+                    return '#72FA77'//Green
+                    
+                }else{
+                    return '#618BEC' //Blue
```

Comment:
Here are the blues that I started to use throughout the site in my last PR. https://coolors.co/ffffff-00171f-003459-007ea7-00a8e8

However, with the current color choices, it looks like there will be good contrast and distinction among the nodes so feel free to proceed with what you have.

Reply:
Scanned through all the changes and it looks good and functions.